src/tree/render.js

```diff
@@ -189,5 +199,5 @@ export function createContainer({
     setExpandedState(newExpandedState);
   }
 
-  return { svg, divBox, allNodes, positioning, width, height, element, zoomWrapper, tooltip };
-}
+  return setContainerData({ svg, divBox, allNodes, positioning, width, height, element, zoomWrapper, tooltip, homeButton });
```

Comment:
Is this just refactoring or do we need to call it here?

Reply:
So we need to set the container data again also when we click on the home button. We could set it there, but one place seems to be enough I would say

Reply:
I was going off Niek's draft, haven't tested it with how it was originally. @niekvanstaveren do you know if it was necessary?

Reply:
So the biggest issue here is the D3 zooming again. Because it stores the zoom state inside the element we need to clear the whole thing from the DOM. This is why we call createContainer from the onClick handler. You could set this up in the index file instead (which also places the button later on the DOM), but then we are putting more things in the index file again. Another option would be to use a fake state for this and change that state whenever you click on the button and add this state to the useEffect.

Reply:
just moving the button creation to before the tooltip creation made it possible to remove z-index, at least

Reply:
But we were running `setContainerData` in index, it would not be adding anything?

Reply:
Yes we were, and now we also need to setContainerData when you click on the button and run the function again